File: 프로그래머스/lv2/12939. 최댓값과 최솟값/최댓값과 최솟값.cpp
#include <string>
#include <iostream>
#include <vector>
#include <sstream>
#include <algorithm>
using namespace std;
vector<string> split(string input,char delimiter){
    vector<string> answer;
    stringstream ss(input);
    string temp;
    
    while(getline(ss,temp,delimiter)){
        answer.push_back(temp);
    }
    return answer;
}
string solution(string s) {
    string answer = "";
    vector<string> st = split(s,' ');
    int tempst1 = stoi(st[0]);
    int tempst2 = stoi(st[0]);
    for(int i = 0;i<st.size();i++){
        tempst1 = min(tempst1,stoi(st[i]));
        tempst2 = max(tempst2,stoi(st[i]));
    }
    answer = to_string(tempst1) + ' ' + to_string(tempst2);
    return answer;
}

File: 프로그래머스/lv2/12939. 최댓값과 최솟값/최댓값과 최솟값.py
def solution(s):
    answer = ''
    st = s.split(' ')
    st.sort()
    mini = int(st[0])
    maxi = int(st[0])
    for i in range(len(st)):
        mini = min(int(st[i]),mini)
        maxi = max(int(st[i]),maxi)
    answer += str(mini) + ' ' + str(maxi)
    return answer